WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -478,5 +541,7 @@ private extension DashboardViewModel {
     enum Constants {
         static let topEarnerStatsLimit: Int = 5
         static let dashboardScreenName = "my_store"
+        static let orderPageNumber = 1
+        static let orderPageSize = 75
```

Comment:
Suggestion: we can fetch only 1 item to reduce the network resource. We only need to know if the store has any orders.

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/12521/commits/18a5542dc98e277a00c2b2f9bfe4503aaefcd233